doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -680,6 +680,15 @@ and an explanation of the underlying technique.
       behavior for objects that are proofs of a statement in :g:`Prop`. This :term:`flag`
       controls this behavior.
 
+   .. table:: Keep Equalities @qualid
+
+      This :term:`table` specifies a set of inductive types for which proof
+      equalities are always kept by :tacn:`injection`. This overrides the
+      :flag:`Keep Proof Equalities` flag for those inductive types.
```

Comment:
Maybe this is because the documentation of the `Keep Proof Equalities` flag is unclear, but it is not immediately clear to me (as someone who doesn't really know much about this feature) how this overrides the flag. Could the documentation become clearer if the table was presented first (not sure)?

Reply:
I do find the documentation of the flag quite unclear, but I don't think that swapping these two entries is going to be helpful. Actually it's even worse, because the flag doesn't do exactly that precisely because of template polymorphism (and the whacky implementation of the flag).

Reply:
OK, maybe let's open a documentation issue to remember to come back to this in the future.